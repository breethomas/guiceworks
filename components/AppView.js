// @flow
import React from 'react'
import css from '../styles/css'
import View from './View'

const style = css({
  display: 'flex',
  flexFlow: 'column wrap',
  minHeight: '100vh',
})

export default (props: any) => (
  <View className={style} {...props} />
)

