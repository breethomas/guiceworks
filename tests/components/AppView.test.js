import React from 'react'
import renderer from 'react-test-renderer'
import AppView from '../../components/AppView'

describe('AppView', () => {
  let component

  beforeEach(() => {
    component = <AppView />
  })

  it('shows the correct snapshot for the AppView component tree', () => {
    const tree = renderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
