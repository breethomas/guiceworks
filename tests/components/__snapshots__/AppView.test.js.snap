// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppView shows the correct snapshot for the AppView component tree 1`] = `
<div
  className={
    Object {
      "data-css-uev7ww": "",
    }
  }
/>
`;
